aws/cloudformation-templates/base/notebook.yaml

```diff
@@ -32,6 +32,13 @@ Parameters:
   Uid:
     Type: String
 
+  UserPoolId:
+    Type: String
+    Description: The Cognito user pool ID.
+
+  PinpointAppId:
+    Type: String
+
 Resources:
   NotebookInstance:
     Type: AWS::SageMaker::NotebookInstance
```

Comment:
Hi James. I find this happens when the source branch gets out of date with the target (master). The diff does not show true in this case. It might make more sense to do a diff with the merged IVS branch.